db/src/rocksdb.rs

```diff
@@ -32,62 +26,97 @@ impl RocksDB {
                 .expect("Failed to set rocksdb option");
         }
 
-        let inner = Inner {
-            db,
-            cfnames: cfnames.clone(),
-        };
-        RocksDB { inner }
-    }
-
-    fn cf_handle(&self, col: Option<u32>) -> Result<Option<ColumnFamily>> {
```

Comment:
It can be defined as a global function `cf_handle(db, col) -> Result` to avoid the duplicated code in read/insert/delete.